### PR TITLE
fix SimpleQueue issue

### PR DESCRIPTION
**Pull request by saikrishna-1996**

### CODE REVIEW

File: config/experiment/simple_a2c/cartpole.yaml [hunk 1]
```diff
@@ -4,19 +4,3 @@ defaults:
       - simple_a2c
   - override /run: headless_play
```
[the remark this answers is not in the document]
Comment (joshair) [reply config/experiment/simple_a2c/cartpole.yaml:5]: override/run: headless_play is correct in my experience.

[the remark this answers is not in the document]
Comment (lhnguyen102) [reply config/experiment/simple_a2c/cartpole.yaml:5]: I couldn't train a2c on cartpole problem without changing the line 5 this morning. Are you able to train a2c on cartpole with headless_play?

[the remark this answers is not in the document]
Comment (lhnguyen102) [reply config/experiment/simple_a2c/cartpole.yaml:5]: It is really weird. We can discuss further tomorrow at the office.

[the remark this answers is not in the document]
Comment (joshair) [reply config/experiment/simple_a2c/cartpole.yaml:5]: Ha is correct.  That should be simple_a2c.

[the remark this answers is not in the document]
Comment (saikrishna-1996) [reply config/experiment/simple_a2c/cartpole.yaml]: Thanks, fixed!

**Review by lhnguyen102 — APPROVED** [review]

**Pull request merged**